R/pkg/R/context.R

```diff
@@ -123,19 +126,46 @@ parallelize <- function(sc, coll, numSlices = 1) {
   if (numSlices > length(coll))
     numSlices <- length(coll)
 
+  sizeLimit <- as.numeric(
+    sparkR.conf("spark.r.maxAllocationLimit", toString(.Machine$integer.max - 10240)))
+  objectSize <- object.size(coll)
```

Comment:
Since the guess of size could easily be wrong, and writing them into disk is not that bad anyway, should we have a much smaller default value (for example, 100M)?